package Exception_Handling;

//public class ThrowKeyword 
//{
//
//	public static void main(String[] args) 
//	{
//		System.out.println("After everything");
//		throw new ArithmeticException("zero");
//
//	}
//
//}

//case 2
//public class ThrowKeyword 
//{
//
//	public static void main(String[] args) 
//	{
//		System.out.println("After everything");
//		try
//		{
//			throw new ArithmeticException("zero");
//		}
//		catch(ArithmeticException ss)
//		{
//			System.out.println("aa gaya exception");
//		}
//	}
//}
//output  After everything
//aa gaya exception

//case 3
//class Invalid extends RuntimeException
//{
//	Invalid(String str)
//	{
//		super(str);
//	}
//	
//}
//public class ThrowKeyword 
//{
//
//	public static void main(String[] args) 
//	{
//		System.out.println("After everything");
//		int age=Integer.parseInt(args[0]);
//		if(age<0)
//		{
//			throw new Invalid("Wrong age he");
//		}
//	}
//}
//output After everything
//Exception in thread "main" Exception_Handling.Invalid: Wrong age he
//at JAVAPROGRAMS/Exception_Handling.ThrowKeyword.main(ThrowKeyword.java:53)

























